Name: Hide Quick Launch Bar in Launcher
Version: 1.3.5-1
Author: Jason Robitaille
Description: Hides the quick launch bar in the launcher  (MIT license)

--- .orig/usr/lib/luna/system/luna-systemui/app/controllers/bar-assistant.js
+++ /usr/lib/luna/system/luna-systemui/app/controllers/bar-assistant.js
@@ -236,6 +236,13 @@
 			}
 			$('appname').innerHTML = $L("Preferences");
 		}
+		else if (payload.id == "com.palm.launcher"){
+			this.controller.serviceRequest('palm://com.palm.systemmanager', {
+				method: 'systemUi',
+				parameters: {"quicklaunch": false}
+			});
+			$('appname').innerHTML = payload.appmenu || payload.title;
+		}
 		else 
 			$('appname').innerHTML = payload.appmenu || payload.title;
 		$('appname').show();
--- .orig/usr/lib/luna/system/luna-applauncher/app/controllers/global-search-assistant.js
+++ /usr/lib/luna/system/luna-applauncher/app/controllers/global-search-assistant.js
@@ -233,7 +233,7 @@
 			this.launcherHiddenAtFirstKeydown = false;
 		}
 		else {
-			SystemManagerService.showQuickLaunch(true);
+			SystemManagerService.showQuickLaunch(false);
 		}
 	},
 	
--- .orig/usr/lib/luna/system/luna-applauncher/app/controllers/launcher-assistant.js
+++ /usr/lib/luna/system/luna-applauncher/app/controllers/launcher-assistant.js
@@ -8,7 +8,7 @@
 	 * hardcoding the quick launch height until we can find a better way. 
 	 * This value = ql graphic height - the offset from the top (small fade area)
 	 */
-	kQuickLaunchHeight: 67,
+	kQuickLaunchHeight: 0,
 	
 	kPageIndicatorSpacing: 6, 	/* spacing between each page indicator */
 	kPageMargin: 10, /* 10 pixel margin on each side of a page */
@@ -495,7 +495,7 @@
 													this.controller,
 													function() {
 														delete this.appDialog;
-														SystemManagerService.showQuickLaunch(true);
+														SystemManagerService.showQuickLaunch(false);
 														this.globalSearchAssistant.enable(true);
 													}.bind(this));
 
